docs/design-notes/protocols.md

```diff
@@ -93,19 +93,56 @@ protocol. Scylla supports several Scylla-only extensions to this protocol,
 described in [protocol-extensions.md](protocol-extensions.md).
 
 By default, Scylla listens to the CQL protocol on port 9042, which can be
-configured via the `native_transport_port` configuration option.
+configured via the `native_transport_port` configuration option. If set
+to 0, the OS will choose the port randomly (which also applies to every CQL
+port discussed later). To explicitly disable listening on a CQL port one
+should set its value to `~` , or `null`.
 
 Scylla also supports the CQL protocol via TLS/SSL encryption which is
 disabled by default and can be enabled via the `native_transport_port_ssl`
-configuration option (the traditional choice is port 9142). Users
-that want to only support the encrypted CQL protocol can disable the
-unencrypted default CQL protocol by setting the `native_transport_port` to 0.
+configuration option. Traditionally, the secure connections are accepted
+on port 9142, but if `client_encryption_options` are specified and
```

Comment:
You broke the explanation (unless you changed the code to match earlier...). When I wrote this paragraph, native_transport_port_ssl was off by default. For this reason 9142 is "the traditional choice" if you want to set it - but the new wording says "connections are accepted" - which is not true unless you set it that way!

Reply:
@jul-stas would you care to comment?

Reply:
I agree with Nadav, fix is on the way

Reply:
I restored the wording